sbin/mdconfig/mdconfig.c

```diff
@@ -86,9 +87,9 @@ usage(void)
 "       mdconfig -l [-v] [-n] [-f file] [-u unit]\n"
 "       mdconfig file\n");
 	fprintf(stderr, "\t\ttype = {malloc, vnode, swap}\n");
-	fprintf(stderr, "\t\toption = {cache, cluster, compress, force,\n");
-	fprintf(stderr, "\t\t          mustdealloc, readonly, reserve, ro,\n");
-	fprintf(stderr, "\t\t          verify}\n");
+	fprintf(stderr, "\t\toption = {async, cache, cluster, compress,\n");
+	fprintf(stderr, "\t\t          force, mustdealloc, readonly, ro,\n");
+	fprintf(stderr, "\t\t          reserve, verify}\n");
 	fprintf(stderr, "\t\tsize = %%d (512 byte blocks), %%db (B),\n");
 	fprintf(stderr, "\t\t       %%dk (kB), %%dm (MB), %%dg (GB), \n");
```

Comment:
Should delete this space after the , and before the \n

Reply:
> Should delete this space after the , and before the \n

Fixed, thanks!